cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/org/apache/hadoop/fs/s3native/S3NInMemoryFileSystem.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.hadoop.fs.s3native;
+
+/**
+ * NOTE: DELETE THIS CLASS ONCE HADOOP DEPENDENCY IS UPDATED TO 2.6.0
```

Comment:
Lets create a Jira, then convert this note to a ``TODO:`` with the Jira in it. Also, lets remove the upper-casing.